src/hyp3_isce2/__main__.py

```diff
@@ -24,6 +27,12 @@ def main():
     )
 
     args, unknowns = parser.parse_known_args()
+
+    username = os.getenv('EARTHDATA_USERNAME')
+    password = os.getenv('EARTHDATA_PASSWORD')
+    if username and password:
+        write_credentials_to_netrc_file(username, password, append=False)
+
```

Comment:
Does an error occur somewhere if these are not provided? Should we raise an error explicitly as an `else` here?